kDrive/UI/View/Files/FileDetail/ShareLink/UsersAccessTableViewCell.swift

```diff
@@ -69,15 +71,18 @@ class UsersAccessTableViewCell: InsetTableViewCell {
             notAcceptedView.isHidden = false
             externalUserView.isHidden = true
         } else if let team = element as? TeamFileAccess {
-            titleLabel.text = team.isAllUsers ? KDriveResourcesStrings.Localizable.allAllDriveUsers : team.name
-            if let savedTeam = DriveInfosManager.instance.getTeam(id: team.id),
-               let usersCount = savedTeam.usersCount {
-                detailLabel.text = KDriveResourcesStrings.Localizable.shareUsersCount(usersCount)
-            } else {
-                detailLabel.text = nil
+            Task { @MainActor in
```

Comment:
A task, so I can call `axait getFrozenTeam` that will perform the work outside the main thread.